Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-jdk-alpine
+COPY target/uberjar/measure-0.1.0-SNAPSHOT-standalone.jar /srv/measure.jar
+
+EXPOSE 3000
```

Comment:
What about rename this file to reflect where it belongs? For example Dockerfile.clojure?